api/v1/controllers/samples.js

```diff
@@ -57,21 +93,33 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   findSamples(req, res, next) {
+    // Check if Cache is on for Wildcard Sample query
+    if (featureToggles.isFeatureEnabled('cacheGetSamplesWildcard')) {
+      const query = req.query.name;
```

Comment:
so we are only using this cache for wildcard queries on the "name" field? if so, please consider renaming the feature toggle to "cacheGetSamplesByNameWildcard" and updating any relevant comments

Reply:
Yes we decided only on name.